### DISCUSSION TITLE
Encountering timeout when publishing larger

### DISCUSSION THREAD

**Discussion by topolarity** [wasmerio/wasmer#2814] [Q&A]
Hope this is the right place to ask this. I'm attempting to publish a large package (~175 MB raw, but gzips well 😛) to wapm.io

I was able to publish yesterday (https://wapm.io/topolarity/zig), but today I'm hitting a timeout:
```bash
$ wapm publish --dry-run
Successfully published package `topolarity/zig@0.10.0-dev.1108+a045b5507`
[INFO] Publish succeeded, but package was not published because it was run in dry-run mode
$ wapm publish
Error: error sending request for url (https://registry.wapm.io/graphql): operation timed out
```

Is there a file size limit for wapm.io, or is there any way to extend this timeout?

**Comment by jcaesar** [comment]
When I ran into this problem before, it was because `wapm publish` tried to publish all files in the folder, including some unrelated intermediate build output, resulting in a multi-100-MB publish. Try publishing from a folder that only contains the files you want to publish. (Still, 175 MB may be too much.)
